src/components/TermsList.tsx

```diff
@@ -0,0 +1,54 @@
+import { LinkList, LinkListLink, Paragraph } from '@utrecht/component-library-react/dist/css-module';
+// import clsx from 'clsx';
+import type { HTMLAttributes } from 'react';
+import './SessionTable.css';
+
+interface Definition {
+  paragraph: string;
+}
+
+interface Source {
+  name: string;
+  url: string;
+}
+
+export interface Term {
+  term: string;
+  definitions: Definition[];
+  sources?: Source[];
+}
+
+interface TermsListProps extends HTMLAttributes<HTMLTableElement> {
+  terms: Term[];
+}
+
+const DefinitionData = ({ paragraph }: Definition) => (
+  <Paragraph className="term-table__definition">{paragraph}</Paragraph>
+);
+
+const SourceData = ({ name, url }: Source) => <LinkListLink href={url}>{name}</LinkListLink>;
+
+export const TermsList = ({ terms }: TermsListProps) => {
+  return (
+    <div className="terms-list-container">
+      <dl>
+        {terms.map(({ term, definitions, sources }, index) => (
+          <div className="term-list_item" key={index}>
+            <dt>{term}</dt>
+            <dl>
+              {definitions.map((definition, index) => (
+                <DefinitionData key={index} {...definition} />
+              ))}
+              Further reading
+              <LinkList>
+                {sources.map((source, index) => (
+                  <SourceData key={index} {...source} />
+                ))}
+              </LinkList>
```

Comment:
Gezien sources optioneel is, een extra check:

```suggestion
              {sources && sources.length && (
                <>
                  Further reading
                  <LinkList>
                    {sources.map((source, index) => (
                      <SourceData key={index} {...source} />
                    ))}
                  </LinkList>
                </>
              )}
```

Reply:
Prachtig!